resource-sharing-partners/src/main/java/no/sikt/rsp/Connection.java

```diff
@@ -0,0 +1,132 @@
+package no.sikt.rsp;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.Base64;
+import no.sikt.alma.generated.Partner;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public final class Connection {
```

Comment:
Selv ville jeg ha spesialisert til to klasser/interfacer som tilbyr henting av data, feks AlmaDataProvider og LibraryDataProvider eller noe slikt, og skjult i interfacene at de bruker http. Dvs at metodene ikke returnerer HttpResponse, men POJO'er. Vet ikke hvor viktig det er i denne sammenheng da. Connection blir et litt vel generisk nav på en klasse for min smak.

Reply:
fikset